packages/flutter_tools/lib/src/isolated/devfs_web.dart

```diff
@@ -158,7 +171,19 @@ class WebAssetServer implements AssetReader {
       final String path = moduleName.replaceAll('.js', '');
       _modules[name] = path;
       _digests[name] = _webMemoryFS.files[moduleName].hashCode.toString();
+      srcIds[moduleName] = moduleName;
```

Comment:
`srcIds[src]` is always `src`. Can `srcIds` be a set instead?

Reply:
Indeed, done!